test/Microsoft.Extensions.Configuration.AzureKeyVault.Test/AzureKeyVaultConfigurationTest.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.KeyVault;
+using Microsoft.Extensions.Configuration.Test;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Extensions.Configuration.AzureKeyVault.Test
+{
+    public class AzureKeyVaultConfigurationTest
```

Comment:
In addition to these, is there any place reasonable where functional tests against a real azure key vault could be run?